packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -188,6 +191,15 @@ export function generateStack(): string | undefined {
 
     if (hasErrorInstanceId(innerError)) {
         newError.addTelemetryProperties({ innerErrorInstanceId: innerError.errorInstanceId });
+    } else {
+        newError.addTelemetryProperties({ untrustedOrigin: 1}); // To match normalizeError
+    }
+
+    // Preserve fluidErrorCode if it exists.
+    // Note that addTelemetryProperties() will not overwrite property if it exsits.
+    // Should be consolidated above via isFluidError() check once isValidLegacyError / patchWithErrorCode are deprecated
+    if (typeof(innerError?.fluidErrorCode) === "string") {
+        (newError as any).fluidErrorCode = innerError?.fluidErrorCode;
```

Comment:
See above comment about what `wrapError` is for IMO.  I don't think it should bring up the inner fluidErrorCode.